### DISCUSSION TITLE
Release v1.5.0 Wish List

### DISCUSSION THREAD

**Discussion by richardcase** [kubernetes-sigs/cluster-api-provider-aws#3542] [General]
Lets capture all the PRs that are open that we want to get in the next feature release **v1.5.0**.  If there are any open PRs that you want included please comment here.

**Comment by richardcase** [comment]
I would like to see the following:

- [ ] #3374
- [ ] #3518
- [ ] #3341
- [ ] #3094

**Comment by mweibel** [comment]
would be great to have https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/3253 in - but requires a CAPI patch release still.

**Comment by dlipovetsky** [comment]
Also #3483 and #3579, both small changes related to the bastion security group.
